packages/swirl-components/src/components/swirl-pdf-reader/swirl-pdf-reader.css

```diff
@@ -154,10 +166,48 @@
 }
 
 .pdf-reader__content {
+  display: flex;
   width: 100%;
   height: calc(100% - 4rem);
 }
 
+.pdf-reader__thumbnails {
+  display: flex;
+  overflow-x: hidden;
+  overflow-y: auto;
+  max-width: 0;
+  height: 100%;
+  padding-top: var(--s-space-16);
+  padding-bottom: var(--s-space-16);
+  flex-basis: var(--thumbnails-width);
+  flex-grow: 0;
+  flex-shrink: 0;
+  align-items: center;
+  background-color: var(--s-background-default);
+  flex-direction: column;
+  transition: max-width 0.2s linear;
+  box-shadow: inset calc(-1 * var(--s-border-width-default)) 0 0 0
+    var(--s-border-default);
+  gap: var(--s-space-4);
+}
+
+.pdf-reader__thumbnail {
+  width: 4rem;
```

Comment:
(question): is 4 correct here? in the design i see 5rem / 80px

Reply:
Fixed.